src/Components/Form/AutoCompleteAsync.tsx

```diff
@@ -102,7 +102,6 @@ const AutoCompleteAsync = (props: Props) => {
               onChange={({ target }) => setQuery(target.value)}
               onFocus={props.onFocus}
               onBlur={() => {
-                setQuery("");
```

Comment:
This change alone should be enough right? What's the reason for having a state to make the number of options to be shown?

Reply:
Using state variable allows to dynamically update the number of options. Earlier no. of options were hardcoded that is why only 10 options were showing but now all available facilities will be shown.
I think removing setQuery should be enough.

Reply:
Yup, removing setQuery should be enough. 

The `showNOptions` is deconstructed from the props passed to the components. Let's not have a state to set the default value for it. 

Instead undefined could be treated as showing all available options instead of defaulting to 10.

Reply:
Hi @rithviknishad. Now if showNOptions is undefined then all the facilities will be shown.